lib/active_model/serializer.rb

```diff
@@ -199,16 +199,22 @@ def serializer_from_options(options)
       opts
     end
 
+    def self.serializers_cache
+      @serializers_cache ||= ThreadSafe::Cache.new
```

Comment:
Indeed ```ActiveSupport::TimeZone.zones_map``` in Rails 4.1 is using ```ThreadSafe::Cache```, as you can see [here]9https://github.com/rails/rails/blob/8764ef95dce3cbd0a58186e3cc1da90cc07862eb/activesupport/lib/active_support/values/time_zone.rb#L189).
Okay, I think it's enough to make me feel comfortable about it. cc/ @kurko @JoshSmith @guilleiguaran